### DISCUSSION TITLE
Choreo Progress Update

### DISCUSSION THREAD

**Discussion by AdrianGev** [NorthernForce/Reefscape#13] [General]
Alright, so I know how to make paths and stuff now. And I'm pretty comfortable using all the baseline features on the UI itself.

Could someone get the robot set up so I can get the stuff configured to practice with autos? (or tell me when this will be done by)

I just need to get certain things configured so the paths are optimized and stuff (max velocity, etc.)

**Comment by kcnan** [comment]
Yeah, if no one else wants to, Adrian and I could work on some drivetrain code on the side. (that way we can integrate choreo and have some testing code for my dashboard). Any thoughts @cml1010101, @Daboss-1, @Whiteheatzack, and @Paradox0100?

**Comment by AdrianGev** [comment]
Also I'm like 100% set on Choreo. I am not using PathPlanner ever again.

**Comment by kcnan** [comment]
So, as it turns out ChoreoLib is literally changing **ALL** their code right now, so this might take a _little_ longer than I hoped
